crates/wdl-ast/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+#### Added
+
+* Documentation change to specify what happens when `text()` is called on an empty `LiteralString` ([#351](https://github.com/stjude-rust-labs/sprocket/pull/351)).
```

Comment:
I don't really think this requires a changelog entry to be honest. If you still feel it does, at least adding it as like a `#### Chore` or something.